autoload/test/go/ginkgo.vim

```diff
@@ -0,0 +1,50 @@
+if !exists('g:test#go#ginkgo#file_pattern')
+  let g:test#go#ginkgo#file_pattern = '\v[^_].*test\.go$'
+endif
+let test#go#ginkgo#patterns = {
+  \ 'test': ['\v^\s*It\("(.*)",', '\v^\s*Context\("(.*)",', '\v.*Describe\("(.*)",'],
+  \ 'namespace': [],
+\}
+
+function! test#go#ginkgo#test_file(file) abort
+  if fnamemodify(a:file, ':t') =~# g:test#go#ginkgo#file_pattern
+    if exists('g:test#go#runner')
```

Comment:
Instead of doing this could you check the test file contains `github.com/onsi/ginkgo` which I am assuming has to be imported on all ginkgo tests? [This is what we've done for other runners](https://github.com/janko-m/vim-test/blob/master/autoload/test/php/phpspec.vim#L7)